vowpalwabbit/common/include/vw/common/text_utils.h

```diff
@@ -28,6 +28,25 @@ void tokenize(char delim, VW::string_view s, ContainerT& ret, bool allow_empty =
   if (!s.empty() || (last_space && allow_empty)) { ret.emplace_back(s.substr(0)); }
 }
 
+template <typename ContainerT>
+void tokenize_expensive(char delim, VW::string_view s, ContainerT& ret, bool allow_empty = false)
```

Comment:
This is only used in wasm so just put it in the wasm wrapper lib

Reply:
removed